src/tests/unit/connection.test.js

```diff
@@ -20,7 +20,7 @@ describe('Test oracledb connection module', () => {
   beforeEach(() => {
     configGetStub = sinon.stub(config, 'get')
       .withArgs('dataSources.oracledb')
-      .returns({});
+      .returns({ oracleSources: ['banner'], banner: {} });
```

Comment:
I'd recommend using a fake instance name like `fakeDb` or something similar for testing. By doing this, it'll be guaranteed we stub all necessary functions while testing, instead of passing the tests by coincidence.

Reply:
Nice catch that was a bug